cmd/utils/flags.go

```diff
@@ -1499,7 +1499,11 @@ func SetProxyConfig(ctx *cli.Context, nodeCfg *node.Config, ethCfg *eth.Config)
 					Fatalf("Proxy external facing enodeURL (%s) cannot be private IP.", "proxy external enodeURL", proxyEnodeURLPair[1])
 				}
 			}
-			ethCfg.Istanbul.ProxyConfigs[i] = &istanbul.ProxyConfig{InternalNode: proxyInternalNode, ExternalNode: proxyExternalNode}
+			ethCfg.Istanbul.ProxyConfigs[i] = &istanbul.ProxyConfig{
+				InternalNode: proxyInternalNode,
+				ExternalNode: proxyExternalNode,
+				StatsHandler: i == 0,
```

Comment:
There is a persistent connection between the proxy and celostats.  One option is to have all of the proxies connect to celostats and the validator routes all celostats related traffic to just one of those proxies, but that would mean that celostats will need to handle many more connections.

The other option, which this PR went with, is having the validator operator manually specify which proxy needs to be the one to handle celo stats traffic.  I'll change the flags so that there is an explicit option for that, but I think for the first version, having the designated celostats proxy be manual is not unreasonable.

We could have a future PR where the validator basically automates that.